unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -127,6 +131,32 @@ private void LoadOwnedWereables(UserProfile userProfile)
                          });
     }
 
+    private void QueryNftCollections(string userId)
+    {
+        if (string.IsNullOrEmpty(userId))
+            return;
+
+        DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(userProfile.userId, NftCollectionsLayer.ETHEREUM)
+           .Then((nfts) =>
+           {
+               ownedNftCollectionsL1 = nfts;
+           })
+           .Catch((error) =>
+           {
+               Debug.LogError(error);
+           });
```

Comment:
I think you can compact this without sacrificing much readability.
```suggestion
        DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(userProfile.userId, NftCollectionsLayer.ETHEREUM).Then(nfts => ownedNftCollectionsL1 = nfts).Catch(Debug.LogError);
```

Reply:
Haven't test the build, but codewise looks good! Left a small suggestion

Reply:
You can apply the same to the L2 promise

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/829/commits/88866dbc3429a55ae28e9702446d8001ec5b5918. Thanks Alex!